src/main/java/com/reckue/post/controllers/ResourceController.java

```diff
@@ -24,7 +24,6 @@
 @RequiredArgsConstructor
 public class ResourceController {
```

Comment:
Please, fix name for swagger from `resource-controller` to .
<img width="356" alt="Screenshot 2020-10-17 at 09 34 07" src="https://user-images.githubusercontent.com/32206555/96330175-e978ab80-105b-11eb-9288-d77ddad0dfe2.png">